doc-content/enterprise-only/ai/ai-credit-card-con.adoc

```diff
@@ -0,0 +1,106 @@
+
+In this case, the Decision Task task decided to process Joe's dispute automatically so it followed *Milestone 1: Automatic Chargeback*.
+
+.DMN model
+The following simplified DMN model shows the decision process that is part of the *fraudDispute* *Decision Task* task:
+
+
+image:ai/simple-dmn.png[]
+
+The input variables are *Age* (cardholder age), *Incident Count* (the number of previous disputes for this cardholder), *Cardholder Status*, and *Fraud Amount*.
+
+{PRODUCT} uses input variables in decision tables to support digital decisioning. The decision tables are created by human business analysts. The analyst creates a draft business requirement analysis document or spreadsheet that stakeholders review and approve. The project designer then uses the {CENTRAL} DMN editor to transfer the data in the analysis document to the DMN model. The Fortress Bank Credit Card Dispute process has two decision tables, the *Cardholder Risk Rating* table and the *Dispute Risk Rating* table. The *Cardholder Risk Rating* table contains three input variables:  *Incident Count*, *Cardholder Status*, and *Age*. The *Dispute Risk Rating* table contains the *Cardholder Status* input variable. It calculates the risk of the dispute based on the cardholder status and the amount of the dispute.
+
+image:ai/riskrating.png[]
+
+* *Cardholder Risk Rating*
++
+Joe is a Silver cardholder over the age of 25. He has had more than two previous disputes, so he has a risk rating of 2. If Joe had no previous disputes, he would have a risk rating of 0.
+
+* *Dispute Risk Rating*
++
+Because Joe is a Silver cardholder and the disputed amount is 40 dollars, Joe has a rating of 1 on the *Dispute Risk Rating* table. If the disputed amount was 140 dollars, Joe would have a risk rating of 2.
+
+The following formula, which is implemented as part of the *Process Automatically* final decision in the DMN model, uses the scores from the two decision tables to determine whether to automatically charge back the disputed amount (*Milestone 1*) or whether more investigation is needed (*Milestone 2*).
+[source]
+----
+(Cardholder Risk Rating + Dispute Risk Rating) < 5
+----
+If Joe's overall risk score is less than five, his disputed amount is charged back automatically (*Milestone 1*). If the overall score is five or greater, then his dispute is processed manually (*Milestone 2*).
```

Comment:
Please change risk value with "5" instead of spelling it out. Check all values for consistency.

Reply:
Done